src/gleam/dynamic.gleam

```diff
@@ -962,9 +962,7 @@ pub fn decode2(
 /// > from(#(1, 2.0, "3"))
 /// > |> decode3(MyRecord, element(0, int), element(1, float), element(2, string))
 /// Ok(MyRecord(1, 2.0, "3"))
-/// ```
 ///
-/// ```gleam
```

Comment:
Could you put back these splits in the code blocks please? 🙏

Reply:
This should be now consistent and done:
https://discord.com/channels/768594524158427167/811254431051808778/1041455396818264174